package.json

```diff
@@ -26,8 +26,11 @@
     "grunt-auto-release": "~0.0.2"
   },
   "peerDependencies": {
-    "karma": ">=0.9"
+    "karma": ">=0.9",
+    "dojo": ">=1.10"
```

Comment:
This should probably be `"dojo": ">=1.7"` since this should work at least back to that version, right?